bindings/cron/cron.go

```diff
@@ -27,14 +27,14 @@ import (
 
 // Binding represents Cron input binding.
 type Binding struct {
-	logger        logger.Logger
-	name          string
-	schedule      string
-	parser        cron.Parser
-	runningCtx    context.Context
```

Comment:
Why getting rid of the context?

Reply:
Same reason as above https://github.com/dapr/components-contrib/pull/2191#discussion_r1016806187

Reply:
I see the comment now. I don’t believe the component is broken, however.

in the runtime, input and output bindings have a separate lifecycle.
Input bindings are stopped via cancelation of the context passes to the Read method.
Output bindings are stopped via invocation of the (optional) Close() method.

You shouldn’t use a single context (or cancellation token) for both. They are meant to have two separate cancellation tokens.

as for using a global variable, that’s going to be a problem when you initialize multiple instances of the component. We’re also going to start (very soon) doing work to allow components to be de-initialized dynamically, so we can’t use global variables.

Reply:
I believe that the intent for the output operation here is that the scheduled InputBinding for that component is stopped. 
Essentially OutputBinding operation controlling the execution of an InputBinding. In my opinion this should not be a valid scenario in the way dapr Bindings are defined, i.e. outputBinding should not affect inputBinding. 
Though it is a completely valid scenario for Cron.

Reply:
I agree with @mukundansundar, this doesn't sound like the correct behavior. I don't believe there's another component which allows stopping (but interestingly, not restarting - so then you have no way to restart a cron except crashing the app) the Read operation using an invocation.

Also, the operation isn't even [clearly documented](https://docs.dapr.io/reference/components-reference/supported-bindings/cron/#binding-support).

My vote (if @berndverst agrees) would be to remove the capability and make the cron binding an input binding only. When we add support for dynamic components (it'll come - eventually) people can use that to start/stop cron tasks. And in the meanwhile, if developers want to ignore cron invocations, they can do that in their own app.

Reply:
@ItalyPaleAle I have changed cron to input binding only component for now.

Reply:
Thanks! Please don't forget to update the docs